symbols/overlay29.yml

```diff
@@ -2244,6 +2260,23 @@ overlay29:
         r0: Entity pointer. Must be a monster, otherwise the function does nothing.
         r1: ?
         r2: ?
+    - name: ReevaluateSnatchMonster
+      address:
+        NA: 0x22F9970
+      description: |-
+        Checks if any monsters on the floor has the snatch status and stores the corresponding info
```

Comment:
What happens if there are multiple monsters with the snatch status? This description makes it seem ambiguous.

Reply:
If there are multiple monsters with snatch, it would set it to the last one in the list. When trying to inflict snatch using TryInflictSnatchStatus, it will remove snatch from any other monsters first to prevent multiple from having it. Would you like me to specify that normally multiple monsters can not have snatch, but if multiple do it selects the last one in the list of monsters?

Reply:
Correction: The first one in the list.

Reply:
Yeah I think that extra info would be good